python/chemiscope/adapters.py

```diff
@@ -14,6 +15,12 @@ def frames_to_json(frames):
 
     if HAVE_ASE and isinstance(frames[0], ase.Atoms):
         return [_ase_to_json(frame) for frame in frames]
+    elif HAVE_ASE and isinstance(frames[0], ase.Atom):
+        raise Exception(
+            "expected ase.Atoms, got ase.Atom. "
+            + "Try passing a list of frames instead of a single frame "
+            + "(`[frame]` instead of `frame`)"
+        )
```

Comment:
Why would we want this, opposed to reading the frames as `frames = [frame]`?

Reply:
You mean manually creating the list ourself if the user passes a single frame? I guess we can do this.